Import of Filebrowser configuration in YAML format leads to a panic

### Checklist

- [x] This is a bug report, not a question.
- [x] I have searched on the [issue tracker](https://github.com/filebrowser/filebrowser/issues?q=is%3Aissue) for my bug.
- [x] I am running the latest [FileBrowser version](https://github.com/filebrowser/filebrowser/releases) or have an issue updating.

### Version

```Text
2.55.0
```

### Description

When importing a previously exported YAML configuration file, filebrowser panics. However, doing the same with JSON works fine.

### What did you expect to happen?

Importing YAML configuration works identical to JSON configuration.

### What actually happened?

Filebrowser panics.

### Reproduction Steps

- Download tarball, extract
- Run `filebrowser config init` (works as expected)
- Run `filebrowser config export config.yaml` (works as expected)
- Run `filebrowser config import config.yaml` (panics, see below)

```
2026/01/21 08:45:08 No config file used
2026/01/21 08:45:08 Using database: [REDACTED]/filebrowser.db
panic: interface conversion: interface {} is map[string]interface {}, not map[interface {}]interface {}

goroutine 1 [running]:
github.com/filebrowser/filebrowser/v2/cmd.init.func7(0x7ff78ca2f6c0?, {0xc00012fa60, 0x1, 0xc00012fe50?}, 0xc000045c50)
        /home/runner/work/filebrowser/filebrowser/cmd/config_import.go:73 +0x4d6
github.com/filebrowser/filebrowser/v2/cmd.init.withStore.func30(0xc000116510?, {0xc00012fa60?, 0xc000045c70?, 0x1?}, 0x0?, 0x0?)
        /home/runner/work/filebrowser/filebrowser/cmd/utils.go:198 +0x23
github.com/filebrowser/filebrowser/v2/cmd.init.withStore.withViperAndStore.func54(0x7ff78d08ab20, {0xc00012fa60, 0x1, 0x1})
        /home/runner/work/filebrowser/filebrowser/cmd/utils.go:192 +0x35e
github.com/spf13/cobra.(*Command).execute(0x7ff78d08ab20, {0xc00012fa30, 0x1, 0x1})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.2/command.go:1015 +0xb02
github.com/spf13/cobra.(*Command).ExecuteC(0x7ff78d08b6a0)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.2/command.go:1148 +0x465
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.2/command.go:1071
github.com/filebrowser/filebrowser/v2/cmd.Execute(...)
        /home/runner/work/filebrowser/filebrowser/cmd/cmd.go:5
main.main()
        /home/runner/work/filebrowser/filebrowser/main.go:10 +0x1a
```

In contrast to a configuration file in JSON format :
- Download tarball, extract
- Run `filebrowser config init` (works as expected)
- Run `filebrowser config export config.json` (works as expected)
- Run `filebrowser config import config.json` (works as expected)

### Files

_No response_

--- a/cmd/config_import.go
+++ b/cmd/config_import.go
@@ -3,7 +3,6 @@ package cmd
 import (
 	"encoding/json"
 	"errors"
-	"path/filepath"
 	"reflect"
 
 	"github.com/spf13/cobra"
@@ -68,12 +67,7 @@ The path must be for a json or yaml file.`,
 			return err
 		}
 
-		var rawAuther interface{}
-		if filepath.Ext(args[0]) != ".json" {
-			rawAuther = cleanUpInterfaceMap(file.Auther.(map[interface{}]interface{}))
-		} else {
-			rawAuther = file.Auther
-		}
+		var rawAuther = file.Auther
 
 		var auther auth.Auther
 		var autherErr error

--- a/cmd/utils.go
+++ b/cmd/utils.go
@@ -3,7 +3,6 @@ package cmd
 import (
 	"encoding/json"
 	"errors"
-	"fmt"
 	"io/fs"
 	"log"
 	"os"
@@ -246,33 +245,6 @@ func jsonYamlArg(cmd *cobra.Command, args []string) error {
 		return nil
 	default:
 		return errors.New("invalid format: " + ext)
-	}
-}
-
-func cleanUpInterfaceMap(in map[interface{}]interface{}) map[string]interface{} {
-	result := make(map[string]interface{})
-	for k, v := range in {
-		result[fmt.Sprintf("%v", k)] = cleanUpMapValue(v)
-	}
-	return result
-}
-
-func cleanUpInterfaceArray(in []interface{}) []interface{} {
-	result := make([]interface{}, len(in))
-	for i, v := range in {
-		result[i] = cleanUpMapValue(v)
-	}
-	return result
-}
-
-func cleanUpMapValue(v interface{}) interface{} {
-	switch v := v.(type) {
-	case []interface{}:
-		return cleanUpInterfaceArray(v)
-	case map[interface{}]interface{}:
-		return cleanUpInterfaceMap(v)
-	default:
-		return v
 	}
 }
 

